website/recursion_website/static/css/base_css.css

```diff
@@ -0,0 +1,16 @@
+  body
+  {
+    display: flex;
+    min-height: 100vh;
+    flex-direction: column;
+  }
+  main {
+    flex: 1 0 auto;
+  }
+  .img{
+  	height:70px;

```

Comment:
Question to the creator of pull request and to the request approver - I want to know how we are ensuring responsiveness if we are giving image height fixed as 70 px. Am i missing anything here?

Reply:
The image will be styled by this, is RECursion logo. Because in base.html there is just one image presently, i used `class=img` thats a mistake, i should have used `id=logo`  i will correct it.

Reply:
Cool, but even then the fixed size of 70px should not be there. We should not keep the logo size same in a projector size screen and on mobile screens.

Reply:
Okay bhaiya, i will implement it soon and try to make site search-friendly.